packages/legacy/core/App/hooks/chat-messages.tsx

```diff
@@ -90,9 +90,9 @@ export const useChatMessagesByConnection = (connection: ConnectionRecord): Exten
     const transformedMessages: Array<ExtendedChatMessage> = basicMessages.map((record: BasicMessageRecord) => {
       const role = getMessageEventRole(record)
       // eslint-disable-next-line
-      const linkRegex = /(?:https?\:\/\/\w+(?:\.\w+)+\S*)|(?:[\w\d\.\_\-]+@\w+(?:\.\w+)+)/gmi
+      const linkRegex = /(?:https?\:\/\/\w+(?:\.\w+)+\S*)|(?:[\w\d\.\_\-]+@\w+(?:\.\w+)+)/gim
```

Comment:
Why this change?

Reply:
Good catch, I'm not sure what made this change. I'll revert it.

Reply:
I'm not sure what made this change. Maybe prettier putting them in alphabetical order? In any case, in JS the order of these options does not matter so I'll leave.

Reply:
Confirmed. These were put in alphabetical order on save in VSCode.